app/tests/ble/central/src/main.c

```diff
@@ -260,6 +272,13 @@ static void discover_conn(struct bt_conn *conn) {
     }
 }
 
+static struct read_params;
```

Comment:
Not sure what this is for. It seems to be a forward declaration for a type, but then it's written to as a variable later (and then never read from)?

Reply:
Sorry, I had pushed a totally broken WIP from my mac when I realized I couldn't run our BLE tests there. Just force pushed fixes to actually make these tests sane.